testdata/BUILD

```diff
@@ -244,6 +244,18 @@ container_image(
 #     ],
 # )
 
+container_image(
+    name = "with_unix_epoch_creation_time",
+    base = ":base_with_volume",
+    creation_time = "610203480",
+)
+
+container_image(
+    name = "with_rfc_3339_creation_time",
+    base = ":base_with_volume",
+    creation_time = "1989-05-03T12:58:00Z",
```

Comment:
This kwarg doesn't feel particularly valuable unless its value can be `{SOURCE_DATE_EPOCH}`.  In fact, I wasn't even imagining exposing this, just making `{SOURCE_DATE_EPOCH}` the default when stamping is enabled.

Reply:
My understanding was that {SOURCE_DATE_EPOCH} would be set as an environment variable, which could be substituted in the build attributes, but I'd believe that I misunderstood the conversation in #155 .

I'll do the `stamp`-based solution.

Reply:
I was imagining that people could define their own values, or use `--define` to inject their own values here, without going through `stamp = True` and `SOURCE_DATE_EPOCH`.

Reply:
maybe also update the `README.md` describing the `creation_time` arg?

Reply:
@ixdy Done.